app/overrides/spree/admin/orders/index/add_distributor_and_order_cycle_filter_inputs.html.haml.deface

```diff
@@ -9,5 +9,5 @@
 .field-block.alpha.eight.columns
   = label_tag nil, t(:order_cycles)
   = select_tag("q[order_cycle_id_in]",
-    options_for_select(OrderCycle.managed_by(spree_current_user).map {|oc| [oc.name, oc.id]}, params[:order_cycle_ids]),
+    options_for_select(OrderCycle.managed_by(Spree::User.last).where('order_cycles.orders_close_at is not null').order('order_cycles.orders_close_at DESC').map {|oc| [oc.name, oc.id]}, params[:order_cycle_ids]),
```

Comment:
`Spree::User.last` will pick whatever user that is not necessarily the current user. I don't think it's a good idea.

@KeirOsborn could you please explain?